src/Solnet.Rpc/Models/Transaction.cs

```diff
@@ -0,0 +1,7 @@
+namespace Solnet.Rpc.Models
+{
+    public class Transaction
+    {
+        
```

Comment:
Why is this empty?

Reply:
Had already written that before we figured your structures for blocks would be useful for getTransaction and you would get it done in the same PR. Removed.